### ISSUE TITLE
Multiple tags with one containing `:`s

### ISSUE THREAD

**Issue by akarl** [segmentio/golines#15]
The below code works as expected:

```go
package pkg

type MyStruct struct {
	Field1 string `url:"http://username:password@foo.bar:1234"`
}
```

No changes:

```shell
golines --dry-run ./pkg/foo.go
// No output
```

The code below does NOT work as expected if I add another tag in front of or behind the `url` tag:

```go
package pkg

type MyStruct struct {
	Field1 string `param:"FOOBAR" url:"http://username:password@foo.bar:1234"`
}
```
```diff
$ golines --dry-run ./pkg/foo.go

--- ./pkg/foo.go
+++ ./pkg/foo.go.shortened
@@ -1,6 +1,6 @@
 package pkg
 type MyStruct struct {
-	Field1 string `param:"FOOBAR" url:"http://username:password@foo.bar:1234"`
+	Field1 string `param:"FOOBAR" url:"http://username:password@foo.bar:1234" http:"" username:"" bar:""`
 }
```

Also if the command above does align the tags, what does adding `--reformat-tags` do?

**Comment by yolken-segment** [comment]
Thanks for the report! This looks like a bug when the tag values contain colons. I'll dig in more in a bit.

**Comment by yolken-segment** [comment]
This should be fixed by https://github.com/segmentio/golines/pull/16. Also note that the reformatting tags behavior is on by default, so adding `--reformat-tags` doesn't do anything- however, you can turn it off by running with `--no-reformat-tags`.

**Comment by telemachus** [comment]
> This should be fixed by #16. Also note that the reformatting tags behavior is on by default, so adding `--reformat-tags` doesn't do anything- however, you can turn it off by running with `--no-reformat-tags`.

@yolken-segment So, I just noticed that the usage for `golines` is a bit confused as far as this flag goes. The usage prints out `--reformat-tags      Reformat struct tags`, which implies that the default is *not* to reformat tags. In addition, there's nothing in the usage to suggest that `--no-reformat-tags` is available.

I'm guessing that this is all handled by kingpin, and I am not (yet) familiar with that library. But if I can figure out how, would you be interested in a pull request to make the usage message clearer? Thanks.

**Comment by yolken** [comment]
Sure, feel free to make any improvements here!
